### PR TITLE
zh trans pod-disruption-budget.md

### PR DESCRIPTION
**Pull request by congfairy**
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Remember to delete this note before submitting your pull request.
>
> For pull requests on 1.17 Features: set Milestone to 1.17 and Base Branch to dev-1.17
> 
> For pull requests on Chinese localization, set Base Branch to release-1.15
>
> For pull requests on Korean Localization: set Base Branch to dev-1.16-ko.\<latest team milestone>
>
> If you need Help on editing and submitting pull requests, visit:
> https://kubernetes.io/docs/contribute/start/#improve-existing-content.
>
> If you need Help on choosing which branch to use, visit:
> https://kubernetes.io/docs/contribute/start#choose-which-git-branch-to-use.
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>

### CODE REVIEW

File: content/zh/docs/reference/glossary/pod-disruption-budget.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+-->
+[Pod 中断预算](/docs/concepts/workloads/pods/disruptions/)允许应用程序所有者为复制的应用程序创建一个对象，以确保具有指定标签的一定数量或百分比的Pod不会在某些时间点被自愿驱逐。 PDB 不能防止非自愿中断，但会计入预算。
+---id: pod-disruption-budgettitle: Pod Disruption Budgetfull-link: /docs/concepts/workloads/pods/disruptions/date: 2019-02-12short_description: > An object that limits the number of {{< glossary_tooltip text="Pods" term_id="pod" >}} of a replicated application, that are down simultaneously from voluntary disruptions.aka: - PDBrelated: - pod - containertags: - operation---<!-- A [Pod Disruption Budget](/docs/concepts/workloads/pods/disruptions/) allows an application owner to create an object for a replicated application, that ensures a certain number or percentage of Pods with an assigned label will not be voluntarily evicted at any point in time. PDBs cannot prevent an involuntary disruption, but will count against the budget.-->[Pod 中断预算](/docs/concepts/workloads/pods/disruptions/)允许应用程序所有者为复制的应用程序创建一个对象，以确保具有指定标签的一定数量或百分比的Pod不会在某些时间点被自愿驱逐。 PDB 不能防止非自愿中断，但会计入预算。
```
Comment (xiangpengzhao) [inline content/zh/docs/reference/glossary/pod-disruption-budget.md]: 这一行的内容是？

File: content/zh/docs/reference/glossary/pod-disruption-budget.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+---
+id: pod-disruption-budget
+title: Pod Disruption Budget
+full-link: /docs/concepts/workloads/pods/disruptions/
+date: 2019-02-12
+short_description: >
+ An object that limits the number of {{< glossary_tooltip text="Pods" term_id="pod" >}} of a replicated application, that are down simultaneously from voluntary disruptions.
```
Comment (xiangpengzhao) [inline content/zh/docs/reference/glossary/pod-disruption-budget.md]: 这句话也翻译一下吧 :)

File: content/zh/docs/reference/glossary/pod-disruption-budget.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+---
+id: pod-disruption-budget
+title: Pod Disruption Budget
+full-link: /docs/concepts/workloads/pods/disruptions/
```
Comment (xiangpengzhao) [inline content/zh/docs/reference/glossary/pod-disruption-budget.md]: 这个链接是不是要改成中文文档中对应的链接了？

File: content/zh/docs/reference/glossary/pod-disruption-budget.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+ - pod
+ - container
+tags:
+ - operation
+---
+
+<!--
+ A [Pod Disruption Budget](/docs/concepts/workloads/pods/disruptions/) allows an application owner to create an object for a replicated application, that ensures a certain number or percentage of Pods with an assigned label will not be voluntarily evicted at any point in time. PDBs cannot prevent an involuntary disruption, but will count against the budget.
+-->
+[Pod 中断预算](/docs/concepts/workloads/pods/disruptions/)允许应用程序所有者为复制的应用程序创建一个对象，以确保具有指定标签的一定数量或百分比的Pod不会在某些时间点被自愿驱逐。 PDB 不能防止非自愿中断，但会计入预算。
```
Comment (xiangpengzhao) [inline content/zh/docs/reference/glossary/pod-disruption-budget.md]: 这里面的链接也改成中文文档中对应的链接吧。另外第二个 Pod 左右加一下空格。

File: content/zh/docs/reference/glossary/pod-disruption-budget.md (hunk 2 above)
Comment (xiangpengzhao) [reply content/zh/docs/reference/glossary/pod-disruption-budget.md]: 翻译后，需要把原文注释掉 :)

File: content/zh/docs/reference/glossary/pod-disruption-budget.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
+---
+id: pod-disruption-budget
+title: Pod Disruption Budget
+full-link: /docs/reference/glossary/
```
Comment (xiangpengzhao) [inline content/zh/docs/reference/glossary/pod-disruption-budget.md]: 不确定是不是应该用这个链接，请 @markthink 确认一下吧

Comment (markthink) [reply content/zh/docs/reference/glossary/pod-disruption-budget.md]: 链接应该是：full-link: /docs/concepts/workloads/pods/disruptions/

**Pull request merged**